### PR TITLE
Code cleaning:

### PR DESCRIPTION
**Pull request by yairelboher**
- shortening functions
- documentation
- fixing variable names
- modularity

there are no changes in the functionality, please write the next code according to the standards above.

### CODE REVIEW

File: app/src/main/java/com/example/sabonit/Account.java [hunk 1]
```diff
@@ -98,20 +105,20 @@ public void setCart(Cart cart)
...
      * Sets the confirmed orders of the account.
-     * @param UID given UID.
+     * @param uid given uid.
      */
-    public void setUID(String UID)
+    public void setUid(String uid)
     {
-        this.UID = UID;
+        this.uid = uid;
     }
```
Comment (liormoalem) [inline app/src/main/java/com/example/sabonit/Account.java:123]: You defined good style,  and specifiec this change, that account write to the db is exactly what we talked about, great!

File: app/src/main/java/com/example/sabonit/Cart.java [hunk 2]
```diff
@@ -57,7 +54,8 @@ public Order getProductOrder(Product product)
     public void addProductToCart(Product product, double liters)
     {
         int index = findProductIndex(product.getFullName());
-        if (index >= 0) {
+        if (index >= 0)
+        {
             double newAmount = ordersList.get(index).setLiters(liters);
             ordersList.get(index).setNewBottle(product.isNewBottle());
```
Comment (liormoalem) [inline app/src/main/java/com/example/sabonit/Cart.java:60]: seems more mesudar and elegant and clean from earlier

**Review by eilonasher — COMMENTED** [review]
I agree with you about the new coding style standards.
No big changes, so you can merge in my opinion

File: app/src/main/java/com/example/sabonit/CartAdapter.java [hunk 3]
```diff
@@ -1,51 +1,52 @@
...
 import androidx.recyclerview.widget.RecyclerView;
-
 import java.text.DecimalFormat;
 import java.text.NumberFormat;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
 import java.util.List;
 
-//******************pizza adapter*******************************/
```
Comment (liormoalem) [inline app/src/main/java/com/example/sabonit/CartAdapter.java:27]: wow! great and sezific work

**Review by liormoalem — COMMENTED** [review]
from me its yes for merging, great changes, high stantard, good, kol hakavod

**Pull request merged**